frontend/src/modules/activity/components/type/activity-type-form-modal.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <app-dialog
+    v-model="isVisible"
+    :title="
+      isEdit ? 'Edit activity type' : 'New activity type'
+    "
+  >
+    <template #content>
+      <section class="px-6 pb-10">
+        <app-form-item
+          class="mb-2"
+          label="Activity type"
+          :validation="$v.name"
+          :required="true"
+          :error-messages="{
+            required: 'This field is required'
+          }"
+        >
+          <el-input v-model="form.name"> </el-input>
+        </app-form-item>
+        <p
+          v-if="form.platform === 'other'"
```

Comment:
This example doesn't seem to be showing in the modal

Reply:
fixed, legacy logic